### DISCUSSION TITLE
Using commands as a spectator?

### DISCUSSION THREAD

**Discussion by o777l** [winstxnhdw/lc-hax#198] [Ideas]
Managed to trigger some landmines and enemies using the crosshair as a spectator. Which got me thinking, is there a way to prompt enemies in this mode, when the chat box is not available? (/hate player, /mob player etc.)

I tried the M3 on a mimic and not sure anything was happening.

So not just for enemies but commands in general is my question

**Comment by winstxnhdw** [comment]
There's no chat box when you're a spectator?

**Reply by o777l** [reply]
I have tried but never got it to open in that mode. Thought it was just how the game works, I'll try some more and get back to you

**Reply by winstxnhdw** [reply]
Oh dear, that's worrying. Please send a screenshot as well.

Also, possession only works while you are in `Phantom`. `/hate` is triggered when you `M3` while looking at a player.

**Reply by o777l** [reply]
Ok so it seems to open the chat box in spectate, but I'm typing blind. Can't see the input

**Reply by winstxnhdw** [reply]
That shouldn't be the case. Are you using any mods along with this?

**Comment by GigglingMinister** [comment]
wait I thought you can use chat while spectate?

**Reply by winstxnhdw** [reply]
I recently refactored that part of the code. Is it broken?

**Reply by GigglingMinister** [reply]
I don't know but I will check it later

**Reply by GigglingMinister** [reply]
working just fine to me

**Reply by winstxnhdw** [reply]
Phew, god bless.
